src/utils/soroban/baseTransaction.ts

```diff
@@ -1,12 +1,12 @@
 import { Account, xdr, Networks, TransactionBuilder, Operation } from 'stellar-sdk';
-
-import getFee from './getFee';
+import getConfig from './getConfig';
+import { Networks as network } from '../../constant/network';
```

Comment:
do not import the Networks as `network`. An enum should have their first letted capitalized.